satpy/readers/viirs_sdr.py

```diff
@@ -27,15 +27,23 @@
 # satpy.  If not, see <http://www.gnu.org/licenses/>.
 
 """Interface to VIIRS SDR format
+**********************************
+
+This reader implements the support of VIIRS SDR files as produced by CSPP. It is
```

Comment:
Isn't this also as produced (or made available) by CLASS?

Reply:
yes, indeed, I just assumed they used cspp. I'll fix the docstring.